.eslintrc

```diff
@@ -11,7 +11,7 @@ parserOptions:
 rules:
   comma-dangle: ["error", "never"]
   indent: [2, 2, {SwitchCase: 1, VariableDeclarator: 2}]
-  max-len: [2, 80, 2]
+  max-len: [2, 90, 2]
```

Comment:
This was a driveby from a discussion about having to find creative ways of wrapping components. Please try and keep your code lengths under 80 and definitely under 90 if possible. But this provides a little wiggle room instead of having to break right at 80.